webpack.config.js

```diff
@@ -1,37 +0,0 @@
-const path = require('path');
```

Comment:
Can we get this back please

Reply:
Same as above, can you elaborate why it's needed? I was trying to reduce dependencies and files in the code

Reply:
A webpack config is required to use webpack and build the project...